redfish-exporter/docs/configuration.md

```diff
@@ -0,0 +1,134 @@
+# Configuration Guide
+
+## Environment Variables
+
+The AMD Redfish Exporter can be configured using the following environment variables:
+
+- `UPDATED`: Last update date of the exporter
+- `DESCRIPTION`: Description of the exporter
+- `LISTENER_IP`: IP address to listen on (default: "127.0.0.1")
+- `LISTENER_PORT`: Port to listen on for Redfish events (default: "8080")  (API events?)
+- `USE_SSL`: Enable SSL/TLS (default: "false")
+- `CERTFILE`: Path to SSL certificate file
+- `KEYFILE`: Path to SSL key file
+- `METRICS_PORT`: Port to expose Prometheus metrics (default: "2112")
+- `SLURM_TOKEN`: SLURM authentication token
+- `SLURM_CONTROL_NODE`: SLURM control node address
+- `SUBSCRIPTION_PAYLOAD`: JSON string containing Redfish subscription details
+- `REDFISH_SERVERS`: JSON array of Redfish server configurations
```

Comment:
Under `REDFISH_SERVERS`, they need to populate the slurm node name correctly for the drain to happen:

`REDFISH_SERVERS=[{"ip": "http://<ip>:<port>", "username": "<username>", "password": "<password>", "loginType": "Session", "slurmNode": "Node1"}]`

Reply:
Added a note to `configuration.md` to call this out